benchmarks/nel/requirements.txt

```diff
@@ -1,3 +1,7 @@
 pyyaml
+requests
 tqdm
-prettytable
\ No newline at end of file
+prettytable
+scikit-learn
+fuzzyset2
+spacyfishing
```

Comment:
`datasets` seems to be required but is not included here?

Reply:
Hm, why do you think it's required? `datasets` is a local package (`nel.scripts.datasets`).

Reply:
I got this error running `preprocess`:

```
================================= preprocess =================================
Running command: unzip -q -o -d assets/reddit assets/reddit.zip
Running command: /mnt/pool/code/spacy-projects/benchmarks/nel/env/bin/python ./scripts/clean_data.py nel.cfg
Traceback (most recent call last):
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/./scripts/clean_data.py", line 20, in <module>
    typer.run(main)
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/env/lib/python3.10/site-packages/typer/main.py", line 864, in run
    app()
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/env/lib/python3.10/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/env/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/env/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/env/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/env/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/env/lib/python3.10/site-packages/typer/main.py", line 500, in wrapper
    return callback(**use_params)  # type: ignore
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/./scripts/clean_data.py", line 16, in main
    Dataset.generate_dataset_from_id(dataset_name).clean_assets()
  File "/mnt/pool/code/spacy-projects/benchmarks/nel/scripts/datasets/dataset.py", line 416, in generate_dataset_from_id
    importlib.import_module(module_name), inspect.isclass
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'datasets.nel'
```

And installing `datasets` made it go away, though I guess that was just an accident because of the name collision? Further steps in the pipeline ran OK afterwards too.

Reply:
Thanks for the log! That was actually a misconfiguration in `project.yml` that should be fixed now. Can you give it another shot without `datasets`?